src/page/shipping-step.page.ts

```diff
@@ -5,8 +5,8 @@ export class ShippingPage {
   private checkBox: ElementFinder;
 
   constructor () {
-    this.nextButton = $('#form > p > button > span');
-    this.checkBox =  $('#cgv');
+    this.nextButton = $('.cart_navigation button[name="processCarrier"]');
+    this.checkBox =  $('.order_carrier_content input#cgv');
```

Comment:
Why is this a better solution? I mean, why do you put .order_carrier_content input before #cgv when #cgv was already an unique locator?

Reply:
I was thinking about improving speed through saving the selector some search time by giving him kind of a path to follow. After researching, i couldn't find anything about how the selectors deeply work that ensures it really saves searching time, plus i realized it wouldn't even be a notable improvement and it affects mantainability, so i'll just leave it as #cgv